packages/common/src/hooks/useSortBy/useSortBy.ts

```diff
@@ -1,28 +1,26 @@
-import { ObjectWithStringKeys } from './../../types/utility';
 import { useCallback, useState } from 'react';
 
-export interface SortRule<T extends ObjectWithStringKeys> {
+export interface SortRule<T> {
   key: keyof T | string;
```

Comment:
also realising for the same reason as filtering, that sorting shouldn't be restricted to the keys on an object. I've made #469 previously to try and sort this out.

Reply:
same as filtering - there's a UI problem when sorting by something that isn't shown; that can be confusing for users. maybe if we have complex sorting options then we will need something like that